tuned/consts.py

```diff
@@ -129,15 +135,29 @@
 CFG_FUNC_LOG_FILE_COUNT = "getint"
 # default log file max size
 CFG_DEF_LOG_FILE_MAX_SIZE = 1024 * 1024
-
+# default listening on dbus
+CFG_DEF_DBUS = True
+CFG_FUNC_DBUS = "getboolean"
+# default listening on unix socket
+CFG_DEF_UNIX_SOCKET = True
+CFG_FUNC_UNIX_SOCKET = "getboolean"
+# default unix socket paths
+CFG_DEF_UNIX_SOCKET_PATHS = "/tmp/tuned_socket"
+CFG_DEF_UNIX_SOCKET_SIGNAL_PATHS = ""
+# default unix socket permissions
+CFG_DEF_UNIX_SOCKET_PERMISIONS = 0o444
+CFG_FUNC_UNIX_SOCKET_PERMISIONS = "getint"
```

Comment:
What is it used for?

Reply:
This is default value and get function describing type of option. See https://github.com/redhat-performance/tuned/blob/420267fa3463ef4a06885847c7ca4cd24c781dc8/tuned/utils/global_config.py#L28